torch/_refs/nn/functional/__init__.py

```diff
@@ -59,6 +61,60 @@
 
 Tensor = torch.Tensor
 
+
+def _dropout_helper(
+    self: TensorLikeType,
+    val: float,
+) -> TensorLikeType:
+    """
+    Helper function for all dropout-type operators. During training,
+    some of the elements of the input tensor are randomly masked.
+
+    Returns the masked tensor of the boolean values.
+
+    """
+
+    return torch.distributions.Uniform(low=0.0, high=1.0).sample() < val
```

Comment:
Ug, I forgot we don't have `torch.uniform`. You were correct this should call `refs.uniform` because that function is missing. My mistake.

https://github.com/pytorch/pytorch/blob/fcc28834765fc4dbff85f8b3992f8a72fc739694/torch/_refs/__init__.py#L4535

Reply:
Updated :)